AnnoyancesFilter/sections/popups.txt

```diff
@@ -80,7 +80,8 @@
 !
 ! Specific sites
 !
-retro-bit.com#%#document.cookie = "empopup=null";
+retro-bit.com##.modals-wrapper.popup-newsletter
```

Comment:
Must be hidden by #$# rule or else user will not have scrolling when using a product which does not support #$# rules.

Reply:
and `.modals-wrapper` is unnecessary